gammapy/irf/psf/map.py

```diff
@@ -16,6 +17,25 @@
 __all__ = ["PSFMap", "RecoPSFMap"]
 
 
+PSF_MAX_OVERSAMPLING = 4
+
+
+def _psf_upsampling_factor(psf, geom, position, energy=None, precision_factor=12):
+    if energy is None:
+        energy = geom.axes[psf.energy_name].center
+    psf_r68 = psf.containment_radius(
+        0.68, geom.axes[psf.energy_name].center, position=position
+    )
+    psf_r68 = np.percentile(psf_r68, 50)
```

Comment:
In fact, a mediane of the psf_r68 is computed, is not it? maybe you can rename the variable...

Reply:
added